CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## Unreleased
+
+**Implemented enhancements:**
+
+- Add support for specifying UPDATE_OPTIONS and ROLLBACK_OPTIONS [\#77](https://github.com/djmaze/shepherd/pull/77) ([moschlar](https://github.com/moschlar))
```

Comment:
Did you write this manually? Thanks, but this is not needed as I am generating the changelog using *github_changelog_generator*. :)